Sources/Navigator/EPUB/Assets/Static/scripts/readium-fixed.js

```diff
@@ -4212,6 +4212,91 @@ function initializeAccessibility(doubleTapLabel, energyBarLabel) {
       element.setAttribute("role", "button")
       element.setAttribute("aria-label", text)
     }
+    
+    /*
+     Given a locator, it will look for decorations (annotations + shared annotations) associated with that locator, so
+     it will animate the found HTML decorator, so, it will move to the right, then it will return to original position
+     */
+    function animateDecorationsFromLocator(locator) {
+        let sharedAnnotationsElements = getSharedAnnotationsItemsForLocator(locator)
+        let annotationsElements = getAnnotationsItemsForLocator(locator)
+        let elements = sharedAnnotationsElements.concat(annotationsElements)
+        elements.forEach(element => {
+            animate(element, 0, true);
+        });
+    }
+
+    function getSharedAnnotationsItemsForLocator(locator) {
+        var elements = []
+        let foundGroupSharedAnnotation = window.readium.getDecorations('sharedAnnotation')
+        if (!foundGroupSharedAnnotation) {
+            return elements
+        }
+        
+        // Search for items in Shared Annotations
+        const itemCount = foundGroupSharedAnnotation.items.length
+        for (var i = 0; i < itemCount; i++) {
+            const item = foundGroupSharedAnnotation.items[i]
+            if (locator.text.highlight === item.decoration.locator.text.highlight) {
```

Comment:
I think we could/should make this stricter by comparing the `before` and `after` of the locators, just in case

Reply:
Fixed